intranet/apps/polls/models.py

```diff
@@ -94,6 +94,28 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        if self.groups.exists():
+            q_objects = Q()
+            for g in self.groups.all():
+                q_objects |= Q(name=g)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = get_user_model().objects.none()
+            for g in groups:
+                users = users.union(g.user_set.all())
+            return users.count()
```

Comment:
I know I said that was the last comment, but I think this will work (and it'll be much more efficient if it does):
```suggestion
            return get_user_model().objects.filter(groups__poll_set=self)
```

@theo-o am I getting this right? Should we test on production and make sure they give the same results?

Reply:
Shouldn't we add .count()?

Reply:
Yes, that would be necessary too.

Reply:
```suggestion
    return get_user_model().objects.filter(groups__poll=self).distinct().count()
```

@anonymoose2's suggestion is not completely correct.

Reply:
Update: @theo-o and I experimented. The correct version is `get_user_model().objects.filter(groups__poll=self).distinct().count()`; please update.